### PR TITLE
migrate tutorials to quarto website

### PR DESCRIPTION
**Pull request by VeenDuco**
- Created a quarto website 
- Migrated the tutorials from .md files to .qmd files for better interaction
- Updated README

To have the website online, github pages should be activated with the "docs" folder as reference.

### CODE REVIEW

File: ASReviewLAB.qmd [hunk 1]
```diff
@@ -1,21 +1,19 @@
-Introductory exercise to AI-aided screening using ASReview LAB
-================
-The ASReview Academy Team
-
-<img src="images/Elas_afstudeerhoedje.png" width="50%" style="display: block; margin: auto;" />
+---
+title: "Introductory exercise to AI-aided screening using ASReview LAB"
```
Comment (Rensvandeschoot) [inline ASReviewLAB.qmd:2]:
```suggestion
title: "Introductory exercise to ASReview LAB"
```

File: ASReviewLAB.qmd [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 
 The **goal** of this exercise is to get familiar with AI-aided screening
-by making use of ASReview LAB v1.x. 
+by making use of ASReview LAB v1.x.
 
 You will learn how to install and set up the software, upload data,
 screen records, and export and interpret the results. The exercise will
 guide you through all the steps of AI-aided screening as described in
 the [workflow on
 Read-the-Docs](https://asreview.readthedocs.io/en/latest/about.html#labeling-workflow-with-asreview).
```
Comment (Rensvandeschoot) [inline ASReviewLAB.qmd:15]:
```suggestion
Read-the-Docs](https://asreview.readthedocs.io/en/latest/about.html#labeling-workflow-with-asreview){target="_blank"}.
```

File: ASReviewLAB.qmd [hunk 3]
```diff
@@ -44,19 +42,19 @@ ASReview](https://asreview.nl/download/) on the ASReview-website.
 More detailed installation information, troubleshooting, and
 installation on a server or via a Docker are available on
 [ReadTheDocs](https://asreview.readthedocs.io/en/latest/installation.html#install-asreview).
```
Comment (Rensvandeschoot) [inline ASReviewLAB.qmd:44]:
```suggestion
[ReadTheDocs](https://asreview.readthedocs.io/en/latest/installation.html#install-asreview){target="_blank"}.  
```

File: ASReviewLAB.qmd [hunk 4]
```diff
@@ -84,7 +82,7 @@ You can also run the software via a server, but you need to take care of
 yourself](https://asreview.readthedocs.io/en/latest/installation.html#server-installation)
```
Comment (Rensvandeschoot) [inline ASReviewLAB.qmd:82]:
```suggestion
yourself](https://asreview.readthedocs.io/en/latest/installation.html#server-installation){target="_blank"}
```

File: ASReviewLAB.qmd [hunk 5]
```diff
@@ -123,7 +121,7 @@ to make the decisions.
 More detailed information about setting up a project can be found on
 [ReadTheDocs](https://asreview.readthedocs.io/en/latest/project_create.html#project-information).
```
Comment (Rensvandeschoot) [inline ASReviewLAB.qmd:122]:
```suggestion
[ReadTheDocs](https://asreview.readthedocs.io/en/latest/project_create.html#project-information){target="_blank"}.
```

File: ASReviewLAB.qmd (hunk 2 above)
Comment (Rensvandeschoot) [reply ASReviewLAB.qmd:15]: Can you add this to all hyperlinks? thanks!

**Review by Rensvandeschoot — CHANGES_REQUESTED** [review]

**Comment by VeenDuco** [comment]
@Rensvandeschoot all links now open in new tabs.

**Pull request merged**